packages/sdk-node/src/clerkExpressRequireAuth.ts

```diff
@@ -38,7 +38,12 @@ export const createClerkExpressRequireAuth = (createOpts: CreateClerkExpressMidd
           clerkClient,
           requestState,
         });
-        return handleInterstitialCase(res, requestState, interstitial);
+        if (interstitial.errors) {
+          // TODO(@dimkl): return interstitial errors ?
```

Comment:
@nikosdouvlis WDYT about passing the errors of the interstitial request to the end-user?
We would probably remove this handling if we drop the `remotePrivateInterstitial()`.

Reply:
We haven't removed it yet, since we need to decide if we want to enforce the publishableKey in all our SDKs.
The local interstitial won't ever return an error but if the publishableKey is missing, fetching interstitial from BAPI will be triggered and that could result to the following errors (based on the Go backend):
- 401 due to missing secretKey or invalid Key
- 500